pages/platform/databases/connect-kubernetes-to-managed-mysql/guide.en-asia.md

```diff
@@ -0,0 +1,302 @@
+
+The parameters you need to know are :
+* the db Host, the db port
+
+    Get them from the "General Information tab"
+* the db Name
+
+    Get it from the "Databases" tab, usually "defaultdb"
+* the db User
+
+    Get it from the "Users" tab, usually "avnadmin"
+* the db Password
+
+    Get it after reset it.
+
+![Password reseted](images/connect-kubernetes-to-managed-mysql11.png){.thumbnail}
+
+Now connect to the database with the command
+
+```
+mysql -uavnadmin -pxxxxxxxxxxxxxx -hmysql-xxxxxxxx-xxxxxxxxx.database.cloud.ovh.net -P20184 defaultdb
+```
+
+<pre class="console"><code>root@mysql-client:/# mysql -uavnadmin -pxxxxxxxxxxxxxx -hmysql-xxxxxxxx-xxxxxxxxx.database.cloud.ovh.net -P20184 defaultdb 
+mysql: [Warning] Using a password on the command line interface can be insecure.
+Welcome to the MySQL monitor.  Commands end with ; or \g.
+Your MySQL connection id is 3603
+Server version: 8.0.26 Source distribution
+
+Copyright (c) 2000, 2021, Oracle and/or its affiliates.
+
+Oracle is a registered trademark of Oracle Corporation and/or its
+affiliates. Other names may be trademarks of their respective
+owners.
+
+Type 'help;' or '\h' for help. Type '\c' to clear the current input statement.
+
+mysql></code></pre>
+
+Setup is done, your Managed MySQL database is fully operational, let's go further and use it with Wordpress hosted in Kubernetes.
+
+## Installing the Wordpress Helm chart
+
+For this tutorial we are using the [Wordpress Helm chart](https://github.com/bitnami/charts/tree/master/bitnami/wordpress){.external} found on [Bitnami repository](https://github.com/bitnami/charts/){.external}. The chart is fully configurable, but here we are using the default configuration, with only the minimal set of customization to make it work well on OVHcloud Managed Kubernetes Service.
+
+### Pre-requisites
+
+As described on tutorial [Installing WordPress on OVHcloud Managed Kubernetes](../../../kubernetes/installing-wordpress/#installing-the-wordpress-helm-chart), remove the default storage class and install the new one.
+
+```
+kubectl delete storageclasses.storage.k8s.io csi-cinder-high-speed
+
+kubectl apply -f https://raw.githubusercontent.com/ovh/docs/develop/pages/platform/kubernetes-k8s/fix-persistent-volumes-permissions/files/fixed-cinder-high-speed-storage-class.yaml
+```
+
+### Customizing your install
+
+By default, the Helm chart installs the Wordpress and a MariaDB on the K8s cluster. As you want to use your OVHcloud Managed MySQL, you need to customize the Helm installation by setting the URL, user and password of your database.
+
+In order to customize your install, without having to leave the simplicity of using helm and the Wordpress helm chart, you can simply set some of the [configurable parameters of the WordPress chart](https://github.com/helm/charts/tree/master/stable/wordpress#configuration){.external}. 
+Then you can add it to your `helm install` with the `--set` option (`--set param1=value1,param2=value2`)
```

Comment:
"Then you can add it to your `helm install` command with the `--set` option (`--set param1=value1,param2=value2`)
"